PROBot/Scripting/LuaScript.cs

```diff
@@ -1160,11 +1160,40 @@ private bool MoveToMap(string mapName)
         }
 
         // API: Moves to a random accessible cell of the specified rectangle.
-        private bool MoveToRectangle(int minX, int minY, int maxX, int maxY)
+        // Parameters can be four integers, or a table of four integers.
+        private bool MoveToRectangle(DynValue minX, int minY = -1, int maxX = -1, int maxY = -1)

```

Comment:
I'm well aware of overloads, however they won't work in this situation, at least no way I know of. The lua globals won't let you define the same function more than once with different possible parameters.